TopMovies/Misc/Date+Extensions.swift

```diff
@@ -11,4 +11,19 @@ extension Date {
   func months(from date: Date) -> Int {
     return Calendar.current.dateComponents([.month], from: date, to: self).month ?? 0
   }
+  static func prettyDate(_ dateString: String) -> String {
```

Comment:
How about storing the Date inside of your models instead of strings in your init from DTO?